mappings/net/minecraft/world/ModifiableWorld.mapping

```diff
@@ -12,6 +12,27 @@ CLASS net/minecraft/class_1945 net/minecraft/world/ModifiableWorld
 		ARG 2 drop
 		ARG 3 breakingEntity
 	METHOD method_8652 setBlockState (Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;I)Z
+		COMMENT Updates the blockstate at a position within this world calling all appropriate callbacks and event handlers.
+		COMMENT <p>
+		COMMENT When called on this server, the new blockstate is stored and propagated to clients and listeners as dictated
+		COMMENT by the supplied flags. Note that calling this on the client will update the world locally, but may not see the
+		COMMENT change persisted across loads. It's recommended to check whether running on the client or server before
+		COMMENT interacting with the world in this way.
+		COMMENT <p>
+		COMMENT Accepted values of the flags are a bitset combination of the following:
+		COMMENT
+		COMMENT 0b0000000 // 0 - DEFAULT/NONE - nothing is notified. The state is merely stored without any visual changes
+		COMMENT 0b0000001 // 1 - PROPAGATE_CHANGE - Propagates a change event to surrounding blocks.
+		COMMENT 0b0000010 // 2 - NOTIFY_LISTENERS - Notifies listeners and clients who need to react when the block changes
+		COMMENT 0b0000100 // 4 - NO_REDRAW - Use in conjunction with NOTIFY_LISTENERS to suppress the render pass on clients.
+		COMMENT 0b0001000 // 8 - REDRAW_ON_MAIN_THREAD - Forces a synchronous redraw on clients.
+		COMMENT 0b0010000 // 16 - FORCE_STATE - Bypass virtual blockstate changes and forces the passed state to be stored as-is.
+		COMMENT 0b0100000 // 32 - SKIP_DROPS - Prevents the previous block (container) from dropping items when destroyed.
+		COMMENT 0b1000000 // 64 - IS_MECHANICAL_UPDATE - Signals that this is a mechanical update, usually caused by pistons moving blocks.
+		COMMENT
+		COMMENT @param pos The position of the blockstate to set.
```

Comment:
The params take the comment directally, not like this.

Reply:
Oh.

Okey, let me fix that.

Reply:
That should sort it!

Reply:
“The position to set the blockstate at”

Reply:
@natanfudge Meaning is the same.

Reply:
“The position of the blockstate to set” means there’s already a blockstate in the position, and you’re using that blockstate to set.

Reply:
Yes. And you're replacing it with a new blockstate.

Reply:
I still disagree but won’t continue because this is bikeshedding

Reply:
Pushed a rewording.